jenkins/pipelines/cd/atom-jobs/manifest-multiarch-common.groovy

```diff
@@ -3,32 +3,12 @@ node("delivery") {
         stage("build multi-arch") {
             withCredentials([usernamePassword(credentialsId: 'harbor-pingcap', usernameVariable: 'harborUser', passwordVariable: 'harborPassword')]) {
                 sh """
-            docker login -u ${harborUser} -p ${harborPassword} hub.pingcap.net
-            cat <<EOF > manifest.yaml
-image: ${MULTI_ARCH_IMAGE}
-manifests:
--
-    image: ${ARM64_IMAGE}
-    platform:
-    architecture: arm64
-    os: linux
--
-    image: ${AMD64_IMAGE}
-    platform:
-    architecture: amd64
-    os: linux
-
-EOF
-            cat manifest.yaml
-            curl -o manifest-tool ${FILE_SERVER_URL}/download/cicd/tools/manifest-tool-linux-amd64
-            chmod +x manifest-tool
-            ./manifest-tool push from-spec manifest.yaml
+            printenv harborPassword | docker login -u ${harborUser} --password-stdin hub.pingcap.net
+            export DOCKER_CLI_EXPERIMENTAL=enabled
+            docker manifest create ${MULTI_ARCH_IMAGE}  -a ${AMD64_IMAGE} -a ${ARM64_IMAGE}
```

Comment:
Missing `docker manifest push`, This will result in no updates in the image repository.

Reply:
forgot push